transmonee_dashboard/src/transmonee_dashboard/pages/base_page.py

```diff
@@ -510,31 +495,28 @@ def indicator_card(
     target_and_total_query = get_target_query(data, numors[0], "Sex", sex_code)
     query = query + " & " + target_and_total_query
     # query = "CODE in @indicator & SEX in @sex_code & RESIDENCE in @total_code & WEALTH_QUINTILE in @total_code"
-
     indicator = numors
-
-    # use filtered chached dataset
+    # use filtered cached dataset
     filtered_data = get_filtered_dataset(**selections)
+    # for col in filtered_data.columns:
```

Comment:
these comments are probably for debugging, to eliminate.
Also, there's really not a change here in the code but a rearrangement by creating filtered_data variable first, do you think this is read better like this @lbassil ?

Reply:
I removed them...